cfg/config.go

```diff
@@ -739,5 +741,17 @@ func BindFlags(v *viper.Viper, flagSet *pflag.FlagSet) error {
 		return err
 	}
 
+	flagSet.IntP("write-buffer-size", "", 4194304, "Size of in-memory buffer that is used per goroutine in parallel downloads while writing to file-cache.")
```

Comment:
nit: should we create a const for default buffer-size?

Reply:
this is auto-generated code, we are specifying the default here (and I followed the convention which is used currently i.e. directly specifying the default, which I think is fine) : 
![image](https://github.com/user-attachments/assets/0118ef86-a0b2-429a-b3ba-8514c94c79da)

I will add a comment though that it is 4MiB.